sympy/combinatorics/fp_groups.py

```diff
@@ -76,6 +76,27 @@ def __new__(cls, fr_grp, relators):
     def free_group(self):
         return self._free_group
 
+    @property
+    def relators(self):
+        return tuple(self._relators)
```

Comment:
Would there be any reason why we couldn't have `self.relators` instead of `self._relators` as a plain attribute?